StripePaymentsUI/StripePaymentsUI/Resources/StripePaymentsUI.xcassets/CardCarousel/card_cartes_bancaires.imageset/Contents.json

```diff
@@ -1,7 +1,17 @@
 {
   "images" : [
     {
-      "filename" : "Carte Bancaire.svg",
+      "filename" : "cartesBancairesLight (1).svg",
```

Comment:
remove (1) in filename

Reply:
Done, also swapped these out for the latest images from JJ